secator/tasks/httpx.py

```diff
@@ -83,17 +91,50 @@ def on_init(self):
 
 	@staticmethod
 	def on_item_pre_convert(self, item):
-		for k, v in item.items():
-			if k == 'time':
-				response_time = float(''.join(ch for ch in v if not ch.isalpha()))
-				if v[-2:] == 'ms':
-					response_time = response_time / 1000
-				item[k] = response_time
-			elif k == URL:
-				item[k] = sanitize_url(v)
-		item[URL] = item.get('final_url') or item[URL]
+		try:
+			for k, v in item.items():
+				if k == 'time':
+					response_time = float(''.join(ch for ch in v if not ch.isalpha()))
+					if v[-2:] == 'ms':
+						response_time = response_time / 1000
+					item[k] = response_time
+				elif k == URL:
+					item[k] = sanitize_url(v)
+			item[URL] = item.get('final_url') or item[URL]
+		# Because item can also be a domain
+		except KeyError as e:
+			if not ('url' in e.args):
+				raise e
```

Comment:
I'm not sure this is needed; this is run on item pre-convert so you shouldn't have any secator output type at this point.